phpBB/styles/prosilver/template/display_options.html

```diff
@@ -1,4 +1,4 @@
-<div class="dropdown-container dropdown-container-{S_CONTENT_FLOW_BEGIN} dropdown-button-control sort-tools">
+<div class="dropdown-container dropdown-container-left dropdown-button-control sort-tools">
```

Comment:
Yes, it should displayed on the right, bidi.css will change the float in 
https://github.com/phpbb/phpbb/blob/master/phpBB/styles/prosilver/theme/bidi.css#L97-L100

if this div get classed with `dropdown-container-right` it will displayed left in RTL due to 
https://github.com/phpbb/phpbb/blob/master/phpBB/styles/prosilver/theme/bidi.css#L124-L127